exporter/awsemfexporter/util.go

```diff
@@ -22,45 +22,50 @@ import (
 
 	"go.opentelemetry.io/collector/model/pdata"
 	conventions "go.opentelemetry.io/collector/translator/conventions/v1.5.0"
+	tracetranslator "go.opentelemetry.io/collector/translator/trace"
 	"go.uber.org/zap"
 )
 
 var patternKeyToAttributeMap = map[string]string{
 	"ClusterName":          "aws.ecs.cluster.name",
 	"TaskId":               "aws.ecs.task.id",
 	"NodeName":             "k8s.node.name",
+	"PodName":              "pod",
 	"ContainerInstanceId":  "aws.ecs.container.instance.id",
 	"TaskDefinitionFamily": "aws.ecs.task.family",
 }
 
-func replacePatterns(s string, attrMap pdata.AttributeMap, logger *zap.Logger) string {
+func replacePatterns(s string, attrMap map[string]string, logger *zap.Logger) (string, bool) {
+	success := true
+	foundAndReplaced := true
```

Comment:
Lint says you don't need this assignment

Reply:
Updated.